Protection events/ExploitGuardNetworkProtectionEvents.txt

```diff
@@ -0,0 +1,7 @@
+//Simple query to show the unique network connections that were audited or blocked by ExploitGuard
+// For more questions on this query, feel free to ping @FlyingBlueMonki on twitter or mattegen@microsoft.com via email
+MiscEvents
+| where EventTime > ago(7d)
+| where  ActionType  =~ "ExploitGuardNetworkProtectionBlocked"
+| summarize count(RemoteUrl) by InitiatingProcessFileName, RemoteUrl, Audit_Only=tostring(parse_json(AdditionalFields).IsAudit)
```

Comment:
When `IsAudit`=true, doesn't it mean 'blocked'?
If so, I suggest you change it to `IsBlocked`

Reply:
Not sure what you mean here.  When "IsAudit" = true it means it was not blocked but only audited.